actionview/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-*   Fix a crash in ERB template error highlighting when the error occurs on a
-    line in the compiled template that is past the end of the source template.
+*   Only link back to referer if referer is a `GET` request when passing `:back`
+    to `link_to`.
 
-    *Martin Emde*
+    *Steve Polito*
```

Comment:
Could you revert the unintended deletions here?

Reply:
Oops!